x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/create_rules.ts

```diff
@@ -6,12 +6,12 @@
 
 import { Alert } from '../../../../../../../plugins/alerting/common';
 import { APP_ID, SIGNALS_ID } from '../../../../common/constants';
+import { transformRuleToAlertAction } from '../../../../common/detection_engine/transform_actions';
 import { CreateRuleParams } from './types';
 import { addTags } from './add_tags';
 import { hasListsFeature } from '../feature_flags';
-import { transformRuleToAlertAction } from './transform_actions';
 
-export const createRules = ({
+export const createRules = async ({
   alertsClient,
   actionsClient, // TODO: Use this actionsClient once we have actions such as email, etc...
```

Comment:
Is this TODO/prop still necessary?

Reply:
I'm going to clean up `actionsClient` and the comments afterward :)